### PR TITLE
Onboarding prep

### PR DESCRIPTION
**Pull request by djcorbett96**
Made some significant updates to the blog starter to prepare it for the onboarding flow

### CODE REVIEW

File: .yextrc
Comment (apav-dev) [inline .yextrc:1]: Can you remove this and add to .gitignore?

File: package.json [hunk 1]
```diff
@@ -24,27 +24,22 @@
     "@headlessui/react": "^1.7.15",
     "@heroicons/react": "^2.0.18",
     "@vitejs/plugin-react": "^4.0.2",
-    "@yext/sites-components": "1.0.0-rc.4",
-    "classnames": "^2.3.2",
+    "@yext/studio": "^0.25.0",
+    "class-variance-authority": "^0.7.0",
```
Comment (apav-dev) [inline package.json]: Can you run `npm i cva@npm:class-variance-authority` and replace all your imports with `import { cva } from "cva";`

File: .yextrc
Comment (djcorbett96) [reply .yextrc:1]: done

File: package.json (hunk 1 above)
Comment (djcorbett96) [reply package.json]: done

File: src/components/atoms/HorizontalDivider.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import { cva } from "class-variance-authority";
+import { cn } from "../../utils/cn";
+import { Sizes } from "../../sharedVariants";
+
+const horizontalDividerVariants = cva("w-0.5 rounded-full", {
+  variants: {
+    color: {
```
Comment (apav-dev) [inline src/components/atoms/HorizontalDivider.tsx]: Can you move textColor, bgColor, and height to sharedVariants? From the sharedVariants folder, can you export a type from that file like `export type Color  = // shared colors` just to keep things consistent. Maybe check to see if there are other spots where this is worth doing

Comment (djcorbett96) [reply src/components/atoms/HorizontalDivider.tsx]: done

File: package.json (hunk 1 above)
Comment (apav-dev) [reply package.json]: sorry can you also remove the `class-variance-authority` import? It repeats right now

File: src/components/atoms/Image.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  aspect: "Square" | "Rectangle" | "None";
+  /**
+   * @displayName Size
+   * @tooltip Controls the size of the image
+   */
+  size?: "Small" | "Medium" | "Large";
+}
+
+export const initialProps: ImageProps = {
+  src: "https://a.mktgcdn.com/p/2L6Bp69LgXuwu_TPYmkvWNy24VxmOpmae33-f4fLlM8/1920x1280.jpg",
```
Comment (apav-dev) [inline src/components/atoms/Image.tsx]: Can you have this point to https://placehold.co/1920x1280? Just so this doesn't break if the image is deleted or changed in the yext account

File: src/components/molecules/FeaturedBlog.tsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+   * @displayName Description Size
+   * @tooltip Controls the size of the description text
+   */
+  descriptionSize: TextSizes;
+}
+
+export const initialProps: FeaturedBlogProps = {
+  name: "Blog Post Title",
+  slug: "blog-post-example",
+  src: "http://a.mktgcdn.com/p/G_jLt9S27Dwyvf26th8ZsXNmCqXTgUh9TEuGfJxtkuo/2070x1380.jpg",
```
Comment (apav-dev) [inline src/components/molecules/FeaturedBlog.tsx]: same comment as above regarding image placeholder url

File: package.json (hunk 1 above)
Comment (djcorbett96) [reply package.json]: yep, just did

File: src/components/atoms/Image.tsx (hunk 3 above)
Comment (djcorbett96) [reply src/components/atoms/Image.tsx]: done

File: src/components/molecules/FeaturedBlog.tsx (hunk 4 above)
Comment (djcorbett96) [reply src/components/molecules/FeaturedBlog.tsx]: done - also updated to the placeholder in my layout

**Review by apav-dev — APPROVED** [review]

**Pull request merged**